python/packages/ai/teams/ai/planner/planner.py

```diff
@@ -10,10 +10,10 @@
 
 from botbuilder.core import TurnContext
 
+from teams import AIHistoryOptions
```

Comment:
this will result in a circular import

Reply:
Can you share how circular import happens? pylint does not complains about circular import and I also haven't figured out the circular import manually.

But it indeed requires update, I will change it to `from teams.ai import AIHistoryOptions` instead of import from root module. This is a good catch. I missed this when using auto completion.